src/utils/constants.js

```diff
@@ -23,6 +24,7 @@ const plotNames = {
   FREQUENCY_PLOT: 'Frequency Plot',
   VIOLIN_PLOT: 'Violin Plot',
   DOT_PLOT: 'Dot Plot',
+  TRAJECTORY_ANALYSIS: 'Trajectory Analysis',
```

Comment:
These are used for displaying the title only? Do you think we could call them `plotTitles` then perhaps?

Reply:
yea, it's called `plotNames`. Sounds ok? or titles better?